src/core/Akka/Actor/Scheduler/DateTimeNowTimeProvider.cs

```diff
@@ -10,29 +10,19 @@
 namespace Akka.Actor
 {
     /// <summary>
-    /// TBD
+    /// The default <see cref="ITimeProvider"/> implementation for Akka.NET when not testing.
     /// </summary>
-    public class DateTimeOffsetNowTimeProvider : ITimeProvider, IDateTimeOffsetNowTimeProvider
+    [Obsolete("This class will be removed in Akka.NET v1.6.0 - use the IScheduler instead.")]
+    public class DateTimeOffsetNowTimeProvider : IDateTimeOffsetNowTimeProvider
     {
         private DateTimeOffsetNowTimeProvider() { }
-        /// <summary>
-        /// TBD
-        /// </summary>
+        
         public DateTimeOffset Now { get { return DateTimeOffset.UtcNow; } }
-
-        /// <summary>
-        /// TBD
-        /// </summary>
+        
         public TimeSpan MonotonicClock {get { return Util.MonotonicClock.Elapsed; }}
-
-        /// <summary>
-        /// TBD
-        /// </summary>
+        
         public TimeSpan HighResMonotonicClock{get { return Util.MonotonicClock.ElapsedHighRes; }}
-
-        /// <summary>
-        /// TBD
-        /// </summary>
+        
         public static DateTimeOffsetNowTimeProvider Instance { get; } = new();
```

Comment:
Basically, I removed all references to this type (all of which were contained inside Akka.Delivery and Akka.Cluster.Sharding.Delivery) and preferred to use the `IScheduler`'s implementation of these same methods instead, which are identical on the `HashedWheelTimer` base class.